app/views/tickets/index.html.erb

```diff
@@ -16,7 +16,7 @@
         <div class="row collapse">
           <% params.delete(:controller) %>
           <% params.delete(:action) %>
-          <%= link_to tickets_path(:csv, params), class: 'radius tiny button right mlm no-m' do %>
+          <%= link_to tickets_path(format: :csv, status: params[:status]), class: 'radius tiny button right mlm no-m' do %>
```

Comment:
Make sure that the tickets_path contains all get variables. If I search for 'test' it should still search for 'test' when exporting the tickets.